dsymbol/src/dsymbol/conversion/second.d

```diff
@@ -362,6 +362,9 @@ void resolveAliasThis(DSymbol* symbol,
 		auto parts = symbol.getPartsByName(aliasThis.breadcrumbs.front);
 		if (parts.length == 0 || parts[0].type is null)
 			continue;
+
+		symbol.aliasThisSymbol = parts[0];
```

Comment:
in theory the language supports multiple alias this (I think SDC even wants to support it) - we should probably iterate over the children and pick all IMPORT_SYMBOL_NAME children (or make a new special one / add a flag) to keep support for this with UFCS.

This might also be used inside `version (DCD)` / `version (AutoComplete)` / `version (never)` statements to workaround DCD limitations by alias this-ing multiple things.

Reply:
Im not sure I understand it correctly, but now I have just added all alias this' to the symbol, and am doing the ufcs check only on the first alias this symbol.